### PR TITLE
Notice on how to reset your icon back to normal

### PR DESCRIPTION
**Pull request by jeeftor**
It could be useful for people to know how to reset the icons back to default.

### CODE REVIEW

File: docs/Actions.md [hunk 1]
```diff
@@ -1051,6 +1051,10 @@ badge(custom: "/Users/xxx/Desktop/badge.png") #or
 badge(shield: "Version-0.0.3-blue", no_badge: true)
 ```
 
+**Note** If you want to reset the badge back to default you can use `git checkout -- <path>/Assets.xcassets/`
```
Comment (KrauseFx) [inline docs/Actions.md]: How about this shows is in terms of `fastlane`
```ruby
sh "git checkout -- <path>/Assets.xcassets/"
```

**Comment by ohwutup** [comment]
hey @jeeftor! thanks for submitting the PR, this is a great idea to show users how to make this happen. would you mind just making the change that @KrauseFx mentioned and then we can merge?

File: docs/Actions.md (hunk 1 above)
Comment (jeeftor) [inline docs/Actions.md]: Changes long since made! :+1:

**Comment by ohwutup** [comment]
hey @jeeftor, sorry, i left out one thing. do you mind squashing those commits down to one before we merge?

**Comment by mfurtak** [comment]
Merged as #1447 - thanks for the contribution! :+1:

**Pull request closed without merging**